titans/model/gpt/gpt.py

```diff
@@ -23,6 +23,10 @@
 
 @no_support(['sp', 'moe'])
 class GPT(nn.Module):
+    """
+    The GPT2 Model transformer with a language modeling head on top (linear layer with weights tied to the input
+    embeddings).

```

Comment:
Can you follow the Google style docstring syntax? e.g. adding `Args` and `Returns` if needed.